<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabelPrinter</class>
 <widget class="QMainWindow" name="LabelPrinter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>220</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ETIKATOR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="lab_tab">
       <attribute name="title">
        <string>Lonja</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Especie</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="lab_species_combo">
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>E. encrasicolus</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>S. pilchardus</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>E. encrasicolus (P)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>E. encrasicolus (G)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>S. pilchardus (P)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>S. pilchardus (G)</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Fecha</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDateEdit" name="lab_date">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="date">
           <date>
            <year>2015</year>
            <month>7</month>
            <day>22</day>
           </date>
          </property>
          <property name="displayFormat">
           <string>dd/MM/yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Origen</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="lab_origin_combo">
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>Lonja de Cádiz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Lonja de Sanlúcar</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Lonja de Barbate</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Núm. Inicial</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="lab_start_spinbox">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Núm. Final</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="lab_end_spinbox">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QPushButton" name="print_lab_labels_btn">
          <property name="text">
           <string>Imprimir</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="survey_tab">
       <attribute name="title">
        <string>Campaña</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Núm. Inicial</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="srv_species_combo">
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>E. encrasicolus</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>S. pilchardus</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>E. encrasicolus (P)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>E. encrasicolus (G)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>S. pilchardus (P)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>S. pilchardus (G)</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QPushButton" name="print_srv_labels_btn">
          <property name="text">
           <string>Imprimir</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Campaña</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="srv_haul_spinbox">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="srv_survey_combo">
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>ECOCADIZ-201507</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ECOCADIZ-R-2015</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="srv_end_spinbox">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Pesca</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Especie</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Núm. Final</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="srv_start_spinbox">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="printer_tab">
       <attribute name="title">
        <string>Impresora</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>186</width>
          <height>42</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&lt;center&gt;Printer IP&lt;/center&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>&lt;center&gt;Port&lt;/center&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="printer_ip">
           <property name="text">
            <string>192.168.2.6</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="printer_port">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>9100</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
